### ISSUE TITLE
Not able to download py-cordex

### ISSUE THREAD

**Issue by michmcr** [euro-cordex/py-cordex#211]
I have tried on a number of different servers through conda to install py-cordex however it is continually getting stuck. I installed it with pip but it says that the module does not exist when I try to load it, nor can I find it when I get the list of packages I have installed. I have even tried the source code which does download but it still does not work. Is there some dependancies that I need to download this.

**Comment by larsbuntemeyer** [comment]
Hi @michmcr, thanks for reporting this! I am not sure what the problem might be, it would be helpful, if you could please provide your
* install command, e.g. `pip install py-cordex`?
* os (`linux`, `macos`, `windows`...)
* python version
* environment, e.g., conda or virtualenv?

**Comment by michmcr** [comment]
Hi @larsbuntemeyer I managed to get it working now, dont know what was happening to it, I tried for days and then it suddenly worked. There is however an issue downloading python-cdo which is needed for py-cordex although I am unsure if this is to do with my virtual environment that I have set up. I am using university servers which are linux on python 3.9 and in an virtual environment. I will make a new virtual environment with no other packages downloaded to see if this works.  

Thanks for your help above
